pkgs/development/ocaml-modules/conduit/lwt.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, buildDunePackage, ppx_sexp_conv, conduit, ocaml_lwt }:
+{ stdenv, buildDunePackage, ppx_sexp_conv, sexplib, conduit, ocaml_lwt }:
```

Comment:
This change is dubious.

Reply:
Huh indeed - must be some leftovers from intermediate state